posthog/storage/object_storage.py

```diff
@@ -0,0 +1,40 @@
+from boto3 import client, resource
+from botocore.client import Config
+from botocore.exceptions import EndpointConnectionError
+
+from posthog.settings import (
+    OBJECT_STORAGE_ACCESS_KEY_ID,
+    OBJECT_STORAGE_BUCKET,
+    OBJECT_STORAGE_HOST,
+    OBJECT_STORAGE_PORT,
+    OBJECT_STORAGE_SECRET_ACCESS_KEY,
+)
+
+s3 = resource(
+    "s3",
+    endpoint_url=f"http://{OBJECT_STORAGE_HOST}:{OBJECT_STORAGE_PORT}",
+    aws_access_key_id=OBJECT_STORAGE_ACCESS_KEY_ID,
+    aws_secret_access_key=OBJECT_STORAGE_SECRET_ACCESS_KEY,
+    config=Config(signature_version="s3v4"),
+    region_name="us-east-1",
+)
+
+client = client(
```

Comment:
Do these calls cause any requests to be made to the internet? If so I'd wrap this in a getter.

Reply:
like this https://github.com/PostHog/posthog/pull/9846/commits/3ad75aae7ff33665b352e9b0ddc6124771528506

(I am still making python up as I go along :))